safe/impact_statistics/function_options_dialog.py

```diff
@@ -236,27 +237,36 @@ def build_widget(self, form_layout, name, key_value):
         hide_label = False
 
         if isinstance(key_value, list):
+            def function(z):
```

Comment:
z <-- please use verbose semantic names except in the case of mathematical formulas where variables represent established symbols for a published formula. 'values' might be a good choice here - thanks!